packages/nextjs/src/app-router/server/auth.ts

```diff
@@ -51,6 +50,25 @@ export const auth = (): Auth => {
   return Object.assign(authObject, { protect, redirectToSignIn });
 };
 
+export const auth = (): Auth | Record<string, any> => {
+  require('server-only');
+
+  if (!EPHEMERAL_MODE_AVAILABLE) {
+    return baseAuth();
+  }
+
+  // NOTE: This is a workaround to allow the development environment to use the clerk key
+  // without having to set the environment variables.
+  try {
+    return baseAuth();
+  } catch (e: any) {
+    if (isClerkKeyError(e)) {
+      return initialState();
+    }
+    throw e;
+  }
```

Comment:
This is necessary to allow the initial request w/o keys to proceed right? During the flow to fetch and make the ephemeral keys available

Reply:
Yeah exactly — this may be different with the keys being propagated through the middleware, so will do some testing and we may be able to remove.